src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPTextViewConnectionListener.cs

```diff
@@ -63,11 +121,131 @@ public void SubjectBuffersConnected(ITextView textView, ConnectionReason reason,
             }
 
             RazorLSPTextViewFilter.CreateAndRegister(vsTextView);
+
+            lock (_lock)
+            {
+                _activeTextViews.Add(textView);
```

Comment:
Hmm, we probably want to only do this if the `textView.TextBuffer.IsRazorLSPBuffer`, ditto in disconnected